fendermint/tracing/src/lib.rs

```diff
@@ -38,7 +38,7 @@ macro_rules! emit {
         tracing::event!(
             name: stringify!($event),
             tracing::Level::INFO,
-            { $($field $(= $value)?),* }
+            { event = tracing::field::display(stringify!($event)), $($field $(= $value)?),* }
```

Comment:
Isn't this duplicative with `name`? Happy to approve, but maybe drop a note on why we need both?

Reply:
Thanks, added the docs